### PR TITLE
turbo: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x]  The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 1.1.1

### CODE REVIEW

File: pages/common/turbo.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# turbo
+
+> High-performance Build System for JavaScript and TypeScript Codebases.
+> More information: <https://turborepo.org/docs/reference/command-line-reference>.
+
+- Log in using the default web browser:
```
Comment (EmilySeville7cfg) [inline pages/common/turbo.md:6]:
```suggestion
- Log in using the default web browser with Vercel account:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/turbo.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# turbo
+
+> High-performance Build System for JavaScript and TypeScript Codebases.
```
Comment (EmilySeville7cfg) [inline pages/common/turbo.md:3]:
```suggestion
> High-performance build system for JavaScript and TypeScript codebases.
> See also: `nx`.
```

[Turborepo alternatives](https://www.libhunt.com/r/turborepo)

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/turbo.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/turbo.md:6]:
```suggestion
- Log in using the default web browser with a Vercel account:
```

File: pages/common/turbo.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Link the current directory to a Vercel organization and enable remote caching:
+
+`turbo link`
+
+- Build the current project:
+
+`turbo run build`
+
+- Run a task without concurrency:
```
Comment (EmilySeville7cfg) [inline pages/common/turbo.md:19]:
```suggestion
- Run the task without concurrency:
```

File: pages/common/turbo.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+`turbo run {{task_name}} --concurrency={{1}}`
+
+- Run a task ignoring cached artifacts and forcibly re-execute all tasks:
```
Comment (EmilySeville7cfg) [inline pages/common/turbo.md:23]:
```suggestion
- Run the task ignoring cached artifacts and forcibly re-execute all tasks:
```

File: pages/common/turbo.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+`turbo run {{task_name}} --force`
+
+- Run a task in parallel across packages:
```
Comment (EmilySeville7cfg) [inline pages/common/turbo.md:27]:
```suggestion
- Run the task in parallel across packages:
```

File: pages/common/turbo.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`turbo run {{task_name}} --parallel --no-cache`
+
+- Unlink the current directory from your Vercel organization and disable Remote Caching:
+
+`turbo unlink`
+
+- Generate a Dot graph of a specific task execution (the output file format can be controlled with the filename):
```
Comment (EmilySeville7cfg) [inline pages/common/turbo.md:35]:
```suggestion
- Generate a Dot graph of the specific task execution (the output file format can be controlled with the filename):
```

**Review by EmilySeville7cfg — APPROVED** [review]
Some small enhancements. ;)

File: pages/common/turbo.md (hunk 3 above)
Comment (mfrw) [reply pages/common/turbo.md:19]: I think if we give a specific example of the task then `the` reads better, but if we keep it generic then `a` reads better.
Again, I think it's a matter of choice :)

**Review by mfrw — APPROVED** [review]
LGTM apart from the `a` v/s `the` :)

**Review by blueskyson — APPROVED** [review]

File: pages/common/turbo.md (hunk 3 above)
Comment (sbrl) [reply pages/common/turbo.md:19]: I think this works better with `a` rather than `the` here.

**Review by sbrl — APPROVED** [review]
Looks good to me! I think there `a` is used is fine - there's no need to exchange it for `the` in this case.

**Pull request merged**